lightning/src/util/enforcing_trait_impls.rs

```diff
@@ -36,22 +36,25 @@ pub struct EnforcingChannelKeys {
 	pub inner: InMemoryChannelKeys,
 	pub last_commitment_number: Arc<Mutex<Option<u64>>>,
 	pub revoked_commitment: Arc<Mutex<u64>>,
+	pub disable_revocation_policy_check: bool,
```

Comment:
Maybe use a configuration macro to make this field only public for testing ?

Reply:
This whole file is not really pub because of the feature gate in `mod.rs`, so I wouldn't worry about gating specific things until we ready it for really being pub.